bip-reconcil.mediawiki

```diff
@@ -0,0 +1,297 @@
+<pre>
+  BIP: ???
+  Layer: Peer Services
+  Title: Transaction announcements reconciliation
+  Author: Gleb Naumenko <naumenko.gs@gmail.com>, Pieter Wuille <pieter.wuille@gmail.com>
+  Comments-Summary: ???
+  Comments-URI: ???
+  Status: Draft
+  Type: Standards Track
+  Created: 2010-00-00
+  License: CC0-1.0
```

Comment:
Suggest a separate `License-Code`

Reply:
Every other bip uses this format:

```
$ git grep 'License: CC0-'
bip-0079.mediawiki:  License: CC0-1.0
bip-0084.mediawiki:  License: CC0-1.0
bip-0123.mediawiki:  License: CC0-1.0
bip-0127.mediawiki:  License: CC0-1.0
bip-0135.mediawiki:  License: CC0-1.0
bip-0156.mediawiki:  License: CC0-1.0
bip-0157.mediawiki:  License: CC0-1.0
bip-0158.mediawiki:  License: CC0-1.0
bip-0178.mediawiki:  License: CC0-1.0
bip-0322.mediawiki:  License: CC0-1.0
```

So it seems like they should be fixed up?

Reply:
It's not a question of format. License-Code is a separate field to give included code a different license.

CC0 isn't a very good license for code. For example, to include code in Core, we would want MIT license terms.

Reply:
Oh, I forgot there were different license tags for the text and code